packages/pigment-css-react/tests/theme/fixtures/themed-component.input.js

```diff
@@ -0,0 +1,53 @@
+import * as React from 'react';
+import { styled } from '@pigment-css/react';
+
+const StatRoot = styled('div', {
+  name: 'PigmentState', // The component name
```

Comment:
```suggestion
  name: 'PigmentStat', // The component name
```

I guess it's **stat** and not **state** 😅 This one and all other instances

Reply:
Nice catch! I should go to sleep 😅